### PR TITLE
Change Sponsors area

### PR DESCRIPTION
**Pull request by jdonszelmann**

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -7,28 +7,6 @@ import Sponsor from "../components/sponsor.astro";
 import SponsorTier from "../components/sponsor-tier.astro";
 ---
 
-<script>
```
Comment (jdonszelmann) [inline src/pages/index.astro:10]: old code, wasn't used anyway

File: src/pages/index.astro [hunk 2]
```diff
@@ -121,90 +74,127 @@ import SponsorTier from "../components/sponsor-tier.astro";
...
+        <!--     <div class="sponsors-row"></div> -->
+        <!-- </SponsorTier> -->
+        <!-- <SponsorTier name="NL Community sponsors"> -->
+        <!--     <div class="sponsors-row"></div> -->
+        <!-- </SponsorTier> -->
+        <!-- <SponsorTier name="NL Community supporters"> -->
+        <!--     <div class="sponsors-row"></div> -->
+        <!-- </SponsorTier> -->
+    </Fragment>
+    <Fragment slot="previous-sponsors">
```
Comment (jdonszelmann) [inline src/pages/index.astro:98]: all sponsors moved here together under tier "last-year" for easy formatting

File: src/pages/index.astro [hunk 3]
```diff
@@ -76,43 +55,17 @@ import SponsorTier from "../components/sponsor-tier.astro";
...
+        <!-- <SponsorTier name="Ecosystem partners"> -->
+        <!--     <div class="sponsors-row"></div> -->
+        <!-- </SponsorTier> -->
+        <!-- <SponsorTier name="Headline sponsor"> -->
+        <!--     <div class="sponsors-row"></div> -->
+        <!-- </SponsorTier> -->
+        <!-- <SponsorTier name="Silver sponsors"> -->
+        <!--     <div class="sponsors-row"></div> -->
+        <!-- </SponsorTier> -->
         <SponsorTier name="Organizing partner">
```
Comment (jdonszelmann) [inline src/pages/index.astro:69]: keep organising partners part, cause we know that already

**Review by erikjee — APPROVED** [review]
Looks nice!

Pushed one commit to add regular padding between old sponsors title and the listing

**Pull request merged**